### ISSUE TITLE
Clarification to README.md

### ISSUE THREAD

**Issue by EmilyBjoerk** [kokorin/Jaffree#119]
The readme says it's based on ffmpeg-cli-wrapper but gives me no further information on why or when I should chose this library over ffmpeg-cli-wrapper.

**Comment by kokorin** [comment]
It's not based, but inspired.
Jaffree provides more functionality in comparison to ffmpeg-cli-wrapper.

чт, 11 февр. 2021 г., 23:17 Emily <notifications@github.com>:

**Comment by EmilyBjoerk** [comment]
Let me rephrase, as a user trying to decide which one I should use, the benefits of choosing jaffree are not clear to me by reading the README. It would probably be a good idea if you could provide a short list of reasons in the README why some one might want to use jaffree over ffmpeg-cli-wrapper

**Comment by kokorin** [comment]
The whole readme is such a list. FrameConsumer & FrameSupplier are the killer feature :-)

**Comment by EmilyBjoerk** [comment]
I'm trying to help you make your project more appealing... if you don't want to do that, that's fine I don't care. Just saying, there's opportunity in the README to better sell your project. I'll close this for now as I don't have more time to spend on this.

**Comment by kokorin** [comment]
I will consider your message as gratitude for spending much time on Jaffree
development. Enjoy it.

сб, 13 февр. 2021 г., 13:50 Emily <notifications@github.com>:
